Ryujinx.HLE/HOS/Services/Fs/FileSystemProxy/FileSystemProxyHelper.cs

```diff
@@ -25,7 +27,11 @@ public static ResultCode LoadSaveDataFileSystem(ServiceCtx context, bool readOnl
             try
             {
                 LocalFileSystem       fileSystem     = new LocalFileSystem(savePath);
-                LibHac.Fs.IFileSystem saveFileSystem = new DirectorySaveDataFileSystem(fileSystem);
+
+                Result rc = DirectorySaveDataFileSystem.CreateNew(out DirectorySaveDataFileSystem dirFileSystem, fileSystem);
```

Comment:
```suggestion
                Result resultCode = DirectorySaveDataFileSystem.CreateNew(out DirectorySaveDataFileSystem dirFileSystem, fileSystem);
```
and add an empty line after.